sprint-issue-template.md

```diff
@@ -0,0 +1,32 @@
+# Sprint <% date %>
+
+## Sprint Goals
+
+- 
+
+## Sprint Discussions
+
+[Schedule](https://github.com/ipfs/pm#sprint-discussion-schedule)
+
+Please take notes in a separate pad, if you can, and link it here.
+
+Endeavour         | Lead           | Pad    | Permanent Link
+:---------------: | :------------: | :----: |
+Apps on IPFS      | Juan Benet     |        |
+infrastructure    | Lars Gierth    |        |
+libp2p            | David Dias     |        |
+node-ipfs         | David Dias     |        |
+go-ipfs           | Jeromy Johnson |        |
+testing + ci      | Jeromy Johnson |        |
+protocol + specs  | Juan Benet     |        |
+bitswap ml        | Juan Benet     |        |
+Data Structures   | Juan Benet     |        |
```

Comment:
IRC + github + realname, or at least have a mapping somewhere it's pretty hard at times to map people together if they don't use the same handle everyhwere